packages/grafana-ui/src/types/forms.ts

```diff
@@ -1,4 +1,7 @@
 import { FormContextValues } from 'react-hook-form';
 export { OnSubmit as FormsOnSubmit, FieldErrors as FormFieldErrors } from 'react-hook-form';
 
-export type FormAPI<T> = Pick<FormContextValues<T>, 'register' | 'errors' | 'control' | 'formState' | 'getValues'>;
+export type FormAPI<T> = Pick<
+  FormContextValues<T>,
+  'register' | 'errors' | 'control' | 'formState' | 'getValues' | 'watch'
+>;
```

Comment:
@peterholmberg @dprokop what is going on here? (just curious), is this us only wanting to expose parts of the react-hook-form API?

Reply:
Yes